neuromation/cli/formatters/ftable.py

```diff
@@ -0,0 +1,121 @@
+from dataclasses import dataclass
+from enum import Enum
+from itertools import zip_longest
+from textwrap import wrap
+from typing import Any, Iterator, List, Optional, Sequence, Union
+
+
+__all__ = ["table"]
+__version__ = "0.1"
+
+
+@dataclass
+class ColumnWidth:
+    min: Optional[int] = None
+    max: Optional[int] = None
+
+    def __post_init__(self) -> None:
+        assert not self.min or self.min > 0
+        assert not self.max or self.max > 0
+
+
+class Align(str, Enum):
+    CENTER = "center"
+    LEFT = "left"
+    RIGHT = "right"
+
+
+_align_to_format = {
+    Align.LEFT: str.ljust,
+    Align.RIGHT: str.rjust,
+    Align.CENTER: str.center,
+    None: str.ljust,
+}
+
+
+def table(
+    rows: Sequence[Sequence[str]],
+    widths: Sequence[Optional[Union[int, ColumnWidth]]] = (),
```

Comment:
The type is too generic.
I prefer `Sequence[ColumnWidth]`: `table()` method is our private API not exposed to users.
In such case better to narrow accepted types down to a bare minimum.